src/status.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export */
+function check(input, item, label, setStored, trashIcon, moreIcon) {
+  input.addEventListener('change', () => {
+    if (input.checked) {
+      item.completed = true;
+      label.style.textDecoration = 'line-through';
+      trashIcon.style.display = 'block';
+      moreIcon.style.display = 'none';
+    } else {
+      item.completed = false;
+      label.style.textDecoration = 'none';
+      trashIcon.style.display = 'none';
+      moreIcon.style.display = 'block';
+    }
+    setStored();
+  });
+
+  if (item.completed === true) {
+    input.checked = true;
+    label.style.textDecoration = 'line-through';
+    moreIcon.style.display = 'none';
```

Comment:
Again your text is really clear and easy to read here. 🙂  Though,  do you think there is more repetition here than is necessary?  Do you think you could use toggle with classes and negation to avoid using the if statement?